frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts

```diff
@@ -101,13 +101,15 @@ export class ViewExperimentComponent implements OnInit, OnDestroy {
   }
 
   deleteExperiment() {
-    const dialogRef = this.dialog.open(DeleteExperimentComponent, {
-      panelClass: 'delete-modal',
-      data: { experimentName: this.experiment.name, experimentId: this.experiment.id }
+    const dialogRef = this.dialog.open(DeleteComponent, {
+      panelClass: 'delete-modal'
     });
 
     dialogRef.afterClosed().subscribe(result => {
```

Comment:
@YashilDepani Can we update name of result variable? may be isDeleteButtonClicked?

Reply:
code updated